scst-store/ingress_multi_cluster.md

```diff
@@ -0,0 +1,140 @@
+Ingress support
```

Comment:
Should there be a link to this page on the [TOC page](https://github.com/SebastianOsuna/docs-tap/blob/main/toc.md)?